src/one_dragon_qt/overlay/overlay_window.py

```
@@ -206,28 +148,71 @@ def paintEvent(self, event: QPaintEvent) -> None:
             if rect is None:
                 continue
 
-            base_color = QColor(_VISION_SOURCE_COLOR.get(item.source, item.color or "#bdbdbd"))
+            # 光晕色 = 来源色（ocr 粉 / template 橙 / yolo 青 / cv 绿），主体白色描边
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)